lib/helper.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
 const runScript = require('runscript');
-const REGEX = /^\s*(\d+)\s+(.*)/;
+const isWin = process.platform === 'win32';
+const REGEX = isWin ? /^(.*)\s+(\d+)\s+$/ : /^\s*(\d+)\s+(.*)/;
 
 exports.findNodeProcess = function* (filterFn) {
-  const command = 'ps -eo "pid,command"';
+  const command = isWin ?
+    'wmic Path win32_process Where "Name = \'node.exe\'" Get CommandLine,ProcessId' :
```

Comment:
I’m not familiar with Windows, is wmic a buildin command?

Reply:
Yes, from [Microsoft offical docs](https://docs.microsoft.com/en-us/windows/desktop/wmisdk/wmic#requirements), but I am not sure about Windows XP.

![image](https://user-images.githubusercontent.com/17738556/42384392-6c86c4de-816c-11e8-9cdd-2fab6bb40354.png)

Reply:
It seems good

Reply:
![image](https://user-images.githubusercontent.com/227713/42405440-3aaf4e8e-81c8-11e8-9c8e-de96b6d5ccd2.png)

https://support.microsoft.com/en-us/help/2664203/-invalid-xml-content-error-message-when-you-run-a-wmic-command-in-wind?sd=rss&spid=14019

Reply:
@atian25 It seems some people have the same issue about `Invalid XML content`. I can't figure out why, I have tested the script on 3 windows pc, works fine.

what about `wmic Path win32_process Where "Name = 'node.exe'" Get CommandLine,ProcessId`, output the same error ?

I am going to test it on `appveyor`.

Reply:
worker for most of the window user is better than nothing~